apps/api/src/routes/UserRoutes.ts

```diff
@@ -18,6 +20,20 @@ userRoutes.get('/', async (req, res, next) => {
     }
 });
 
+userRoutes.get('/me', ensureAuthentication, async (req, res, next) => {
```

Comment:
Rename route from "/me" to "/currentUser"?

Reply:
will do!